### DISCUSSION TITLE
Photos timeline

### DISCUSSION THREAD

**Discussion by seboos75** [jellyfin/jellyfin#10050] [General]
i'm starting the switch from plex to jellyfin, i like jellyfin, but... i only see the slider with the alphabet on the photos, is it possible to navigate to photos in the timeline or search for photos between creation dates?

**Comment by Sivwg** [comment]
I'm in same situation of migration to Jellyfin and I'm also looking for this timeline management for Personnal Photos or Vidéos. this is the only weak point i discover in a few days of use.

More than a simple timeline based on one unique EXIF data, it should be nice to get possibility to display pictures base on selected Exif Fields (IE Camera Model, Shot date or Location, shot date......)
